### PR TITLE
Newsletter #11

### PR DESCRIPTION
**Pull request by RichardLitt**
Renamed Weekly to Newsletter.
Added in all comments from https://github.com/ipfs/newsletter/issues/41. 

Needs:
- [ ] Look for and add press.
- [x] Add contributors (cc @eminence)
- [ ] @noffle Can you write something better about twitter-resolver? 
- [ ] Review

### CODE REVIEW

**Comment by hackergrrl** [comment]
_common-readme_: I think it might make sense to drop this from the newsletter? It's not specific to IPFS and I think it's confusing to list both projects here. If you want to link to it within your repo though as a mention that'd be cool.

**Comment by jbenet** [comment]
What period of time does this weekly cover? I will mention various things once i know

**Comment by RichardLitt** [comment]
@jbenet The last couple of weeks; anything since April 25th.

**Comment by eminence** [comment]
Added Contributors for this period (2015-04-25T00:00:01 to 2015-05-16T00:00:01)

Note that I just pushed a small change to get_commits.py that will now use the github API to get the commit authors github login/name.  See 4416c5dfce8a0ba1a809702a609424344f61cb78

Edit: April 2015 was most certainly not the correct data range to use!  This has now been fixed, thanks @dignifiedquire .  `Contributors for this period (2016-04-25T00:00:01 to 2016-05-16T00:00:01)`

File: published/011-2016-May-16.md [hunk 1]
```diff
@@ -82,6 +82,31 @@ Across the entire IPFS GitHub organization, the following people have committed
...
+* [@ehd](https://github.com/ehd) (Stephan Seidt)
+* [@grncdr](https://github.com/grncdr) (Stephen Sugden)
+* [@timgws](https://github.com/timgws) (Tim Groeneveld)
+* [@tv42](https://github.com/tv42) (Tommi Virtanen)
+* [@travisperson](https://github.com/travisperson) (Travis Person)
+* [@vitorbaptista](https://github.com/vitorbaptista) (Vitor Baptista)
+* [@wking](https://github.com/wking) (W. Trevor King)
+* [@bmcorser](https://github.com/bmcorser) (bmcorser)
+* [@krl](https://github.com/krl) (kristoffer)
+
```
Comment (dignifiedquire) [inline published/011-2016-May-16.md]: This list seems very incomplete. @nginnever @diasdavid and me  (@dignifiedquire) have pushed a decent amout of code in the past weeks. Also krl is mentioned twice in this list

Comment (eminence) [inline published/011-2016-May-16.md]: Hmm, yes.  Where are you very important people.  Let me look.  Thanks for paying attention!

Comment (eminence) [inline published/011-2016-May-16.md]: oh my goodness.  i did a terrible thing.  when I ran the tool, I used 2015, not 2016!  A fix is incoming!

Comment (dignifiedquire) [inline published/011-2016-May-16.md]: :D that would explain this list

**Comment by dignifiedquire** [comment]
thanks @eminence!

**Comment by RichardLitt** [comment]
Got another bug in `name-your-contributors` -- not sure what it is, don't have the time to debug it right now. Need to go through and update it before the list at the bottom reflects more than the committers only.

**Comment by RichardLitt** [comment]
@eminence I want to close this newsletter and start fresh with a new one. Can you pull out your commits from this PR and put them in their own?

**Comment by eminence** [comment]
I had 3 commits in this PR, 2 of them related to the newsletter itself, and 1 was a tools change to get_commits.py.  That tools change (4416c5dfce8a0ba1a809702a609424344f61cb78) is already in the master branch, so it's now safe to delete this PR

**Comment by RichardLitt** [comment]
Thanks @eminence. Rebased. 

This newsletter is very old at this point, and the next one should be started anew. I am going to close this and not try to send a retrospective newsletter for this two week session, but rather for the two months or so since the last one.

**Pull request closed without merging**